### DISCUSSION TITLE
IDN-Domains not supported

### DISCUSSION THREAD

**Discussion by ibariens** [immich-app/immich#14644] [Feature Request]
### The bug

using an [IDN-Domain](https://www.denic.de/wissen/idn-domains) is not supported.
e.g. examplé.com would become xn--exampl-gva.com

entering both of these domains raises a FormatExeption

### The OS that Immich Server is running on

iOS

### Version of Immich Server

latest

### Version of Immich Mobile App

1.122.2

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
default
```

### Your .env content

```Shell
default
```

### Reproduction steps

1. Enter Server URL http://examplé.com:2283/api  or http://xn--exampl-gva.de:2283/api

**Comment by mmomjian** [comment]
Converting to feature request, thank you.

**Comment by bo0tzz** [comment]
#12897
